### PR TITLE
Task/remove letters

### PR DESCRIPTION
**Pull request by jzbahrai**
# Summary | Résumé

Remove letter code

### CODE REVIEW

File: app/letters/rest.py [hunk 1]
```diff
@@ -10,6 +8,4 @@
 
 @letter_job.route("/letters/returned", methods=["POST"])
 def create_process_returned_letters_job():
-    references = validate(request.get_json(), letter_references)
-
-    return jsonify(references=references["references"]), 200
+    pass
```
Comment (andrewleith) [inline app/letters/rest.py:11]: Why not get rid of all of this, would that cause other issues?

File: app/letters/utils.py
Comment (andrewleith) [inline app/letters/utils.py:1]: Can we not delete this entire file?

File: app/service/rest.py [hunk 2]
```diff
@@ -793,8 +773,7 @@ def create_one_off_notification(service_id):
 
 @service_blueprint.route("/<uuid:service_id>/send-pdf-letter", methods=["POST"])
 def create_pdf_letter(service_id):
-    resp = send_pdf_letter_notification(service_id, request.get_json())
-    return jsonify(resp), 201
+    pass
```
Comment (andrewleith) [inline app/service/rest.py:776]: Are these routes reachable by end-users? If so, what kind of response is received after this change?

File: app/letters/utils.py
Comment (jzbahrai) [reply app/letters/utils.py:1]: we could! Lets do this incrementally though. I removed a bunch, will keep removing more cause its a lot of finding is something is called elsewhere or not.

File: app/service/rest.py (hunk 2 above)
Comment (jzbahrai) [reply app/service/rest.py:776]: I think they are currently in admin, though not visible. will remove them all in another pr

**Review by andrewleith — APPROVED** [review]
LGTM 🪓

**Pull request merged**